esi_leap/common/policy.py

```diff
@@ -36,10 +39,25 @@
         [{'path': '/contracts', 'method': 'POST'}]),
     policy.DocumentedRuleDefault(
         'esi_leap:contract:get',
-        'rule:is_admin or rule:is_owner',
+        'rule:is_admin or rule:is_owner or rule:is_lessee',
         'Retrieve contract',
-        [{'path': '/contracts', 'method': 'GET'},
-         {'path': '/contracts/{contract_ident}', 'method': 'GET'}]),
+        [{'path': '/offers', 'method': 'GET'},
+         {'path': '/offers/{offer_ident}', 'method': 'GET'}]),
+    policy.DocumentedRuleDefault(
+        'esi_leap:contract:get_all_no_view',
+        'rule:is_admin or rule:is_lessee',
+        'Retrieve all contracts owned by project_id',
+        [{'path': '/contracts', 'method': 'GET'}]),
+    policy.DocumentedRuleDefault(
+        'esi_leap:contract:get_all_owned_view',
+        'rule:is_admin or rule:is_owner',
+        'Retrieve all contracts with offer owner',
+        [{'path': '/contracts', 'method': 'GET'}]),
+    policy.DocumentedRuleDefault(
+        'esi_leap:contract:get_all_all_view',
```

Comment:
Let's rename these policies to match what Ironic uses - esi_leap:contract:list for non-admins, and esi_leap:contract:list_all (assuming we get rid of the get_all_owned_view rule).